qiita_db/processing_job.py

```diff
@@ -292,7 +292,13 @@ def launch_torque(env_script, start_script, url, job_id, job_dir,
 
     # wait for qsub_cmd to finish, but not longer than the number of
     # seconds specified below.
-    q.join(2)
+    init_time = time()
+    q.join(5)
+    total_time = time() - init_time
```

Comment:
Since total_time is the only thing being saved, you could also simply reuse the init_time variable in place of creating a second one (total_time). Although there is something to be said for clarity.

Reply:
yah, rather be clear ...

Reply:
Gotcha.